src/main/java/org/datadog/jmxfetch/AttachApiConnection.java

```diff
@@ -49,7 +52,12 @@ private String getJMXUrlForProcessRegex(String processRegex) throws com.sun.tool
                 return connectorAddress;
             }
         }
-        throw new IOException("Cannot find JVM matching regex: " + processRegex);
+
+         List<String> jvms = new ArrayList<String>();
+         for (com.sun.tools.attach.VirtualMachineDescriptor vmd : com.sun.tools.attach.VirtualMachine.list()) {
+             jvms.add( vmd.displayName() );
```

Comment:
This could be done within the previous loop `for` loop, c.f. https://github.com/DataDog/jmxfetch/pull/100/files#diff-2e06a50bff42ed16d309e1802b58faecR38